dev/CommonStyles/Common_themeresources_any.xaml

```diff
@@ -90,6 +90,7 @@
             <Color x:Key="SolidBackgroundFillColorSecondary">#1C1C1C</Color>
             <Color x:Key="SolidBackgroundFillColorTertiary">#282828</Color>
             <Color x:Key="SolidBackgroundFillColorQuarternary">#2C2C2C</Color>
+            <Color x:Key="SolidBackgroundFillColorTransparent">#00202020</Color>
```

Comment:
The name indicates that this color should be fully transparent, but it isn't?:)
may be change the name to something else?:)